data-plane-framework/draft-ietf-detnet-data-plane-framework.xml

```diff
@@ -224,77 +232,88 @@
         </figure>
        <t>
          The DetNet forwarding sub-layer may be directly provided by the DetNet
-	 service sub-layer, for example by IP tunnels or MPLS. 
-	 Alternatively, an overlay approach
+         service sub-layer, for example by IP tunnels or MPLS. 
+         Alternatively, an overlay approach
          may be used in which the packet is natively carried between key nodes
          within the DetNet network (say between PREOF nodes) and a sub-layer is
          used to provide the information needed to reach the next hop in the
          overlay.
        </t>
        <t>
-         The forwarding sub-layer provides the QoS related functions needed by the
-         DetNet flow. It may do this directly through the use of
-         queuing techniques and traffic engineering methods, or it may do this
-         through the assistance of its underlying connectivity. For example it
-         may call upon Ethernet TSN capabilities defined in IEEE 802.1 TSN
-         <xref target="IEEE802.1TSNTG"/>.
-       </t>
-       <t>
+         The forwarding sub-layer provides the QoS related functions needed by
+         the DetNet flow. It may do this directly through the use of queuing
+         techniques and traffic engineering methods, or it may do this through
+         the assistance of its underlying connectivity. For example it may call
+         upon Ethernet TSN capabilities defined in IEEE 802.1 TSN <xref
+                 target="IEEE802.1TSNTG"/>.  The forwarding sub-layer uses
+         buffer resources for packet queuing, as well as reservation and
+         allocation of bandwidth capacity resources. 
+ </t>
+ <t>
          The service sub-layer provides additional support beyond the
-         connectivity function of the forwarding sub-layer. An example
-         of this is Packet Replication, Elimination, and Ordering
-         functions see <xref target="PREOF"/>.
+         connectivity function of the forwarding sub-layer. An example of this
+         is Packet Replication, Elimination, and Ordering functions see <xref
+                 target="PREOF"/>.  The ordering (POF) uses sequence numbers
+         added to packets enabling a range of packet order protection from
+         simple ordering and dropping out-of-order packets to more complex
+         reordering of a fixed number of out-of-order, minimally delayed
+         packets.  Reordering requires buffer resources and has impact on the
+	 delay and jitter of packets in the DetNet flow. 
+	 These functions are optional and many require sequence numbers that are 
+	 provided by the service sub-layer or available to the service sub-layer from a higher layer

```

Comment:
"available to the service sub-layer from a higher layer" no such option in current data plane drafts.

Reply:
[Don] This is straight from the DetNet Architecture.  The question was is Sequence number available to IP? The general answer is not at IP but maybe with an option or at a higher layer. The framework options do not have to apply only to the data planes we have today.

[Balazs] Ack. No change needed.